### DISCUSSION TITLE
Looking for maintainers!

### DISCUSSION THREAD

**Discussion by ryparker** [ryparker/jest-circus-allure-environment#200] [General]
I haven't been able to spend enough time on this project lately due to a new gig at AWS and moving to Seattle. It's unfortunate because I've noticed a spike in usage and issues recently (which is exciting!). If anyone is interested in maintaining the project please let me know in this thread.

I'd be grateful for any help. As a maintainer all I'd ask is:
1. Limit changes to the inputs/outputs as this would require changes on the user side. 
2. Test changes by running the jest tests (`yarn test`) and inspecting the allure report (`allure serve`) for breaks in functionality.

 @lpolito

**Comment by lpolito** [comment]
I can help, for sure! I suspect I'll be using this more and more, so I can help in any way I can.

**Reply by ryparker** [reply]
Excellent! Thank you!

Send me an email at *** so that I have your email. Just in case I need to shoot you a private message.
